[15.0][IMP] web_widget_many2one_simple: Add a demo view to better test the functionality. (#2350)
OCA/web: web_widget_many2one_simple/README.rst

Usage
=====

> Go to Contacts > Create or edit a record and set "Industry" field (in Sales & Purchase tab).

Reviewer: Put this after the other paragraph indicating that using demo data, the widget can be used this way.

> Only if you set the correct indstry name (Administrative for example) will it be defined, otherwise it will try to create one with that name.

Reviewer 2: ```suggestion
Only if you set the correct industry name (Administrative for example) will it be defined, otherwise it will try to create one with that name.
```

Reviewer: ```suggestion
Only if you set the correct industry name ("Administrative" for example), it will be defined. Otherwise, it will try to create one with that name.
```

You need to declare the usage of the new widget as follows:
``<field name="xxx" widget="many2one_simple" options="...">``

If you have a database with demo data, you can test this widget following these instructions:

> #. Go to Contacts.

Reviewer: Incorrect syntax: missing empty line. This is not how I put the suggestion...

You need to declare the usage of the new widget as follows:
``<field name="xxx" widget="many2one_simple" options="...">``

> Go to Contacts > Create or edit a record and set "Industry" field (in Sales & Purchase tab).

Reviewer: Still missing the explanation about demo data:

```suggestion
If you have a database with demo data, you can test this widget following these instructions:

#. Go to Contacts.
#. Create or edit a record.
#. Set "Industry" field (in Sales & Purchase tab).
...
```
